pkg/ebpf/c/constant_folding.patch

```diff
@@ -18,7 +18,7 @@
 +#error Unknown architecture
 +#endif
 +
-+#define symver_wrap_d1(func)                                    \
++#define symver_wrap_d(func)                                     \
```

Comment:
Could we leave this as `symver_wrap_d1`? The number indicates the number of arguments.